plugins/techdocs/src/home/components/Tables/DocsTable.tsx

```diff
@@ -82,6 +83,13 @@ export const DocsTable = (props: DocsTableProps) => {
     };
   });
 
+  const defaultOptions: object = {
+    paging: true,
+    pageSize: 20,
+    search: true,
+    actionsColumnIndex: -1,
+  };
+
```

Comment:
thanks for reviewing, where would you propose to place the default options on? my reasoning is because i saw the `defaultColumns` and `defaultActions` on the `DocsTable` component, hence i feel its a good idea to have the `defaultOptions` here as well.

Reply:
A-ha, I didn't see the other default options, I agree with you, let's keep it all together 🙂